contrib/test_namespace.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash -eEx
+#
+# Testing script for UCX namespace related functionality
+#
+# Copyright (c) NVIDIA CORPORATION & AFFILIATES, 2023. ALL RIGHTS RESERVED.
+#
+# See file LICENSE for terms.
+#
+#
+# Environment variables set by Jenkins CI:
+#  - WORKSPACE           : path to work dir
+#  - BUILD_NUMBER        : jenkins build number
+#  - JOB_URL             : jenkins job url
+#
+
+realdir=$(realpath $(dirname $0))
+source ${realdir}/../buildlib/tools/common.sh
+
+WORKSPACE=${WORKSPACE:=$PWD}
+ucx_inst=${WORKSPACE}/install
```

Comment:
do we need this?

Reply:
removed, but kept `ucx_inst` to still allow build from within directory which is useful for manual runs on dev